routes/items/item_response.js

```diff
@@ -17,6 +18,37 @@ const filteredItemsQuery = (Item, query, user_id) => ({ items, tree }) => {
   }
 }
 
+const selectedItemsQuery = item_id => (
+  { order: [['createdAt', 'ASC']], where: {
+    $or: [
+      { id: item_id },
+      { parent_id: item_id }
+    ]
+  }, FETCH_ATTRIBUTES }
+)
+
+const filterSelectedItems = (Item, query, user_id) => ({ items, tree }) => {
+  const where = Object.assign( {}, whereSearch( query ), whereCompleted( query), { user_id } )
+
+  if( Object.keys( where ).length === 1 ) {
+    return { items, tree }
+  } else {
+    return Item.findAll({ where, attributes: [ 'id' ] })
+      .then( pruneTree( items, tree ))
+  }
+}
+
+const findBreadcrumbNav = ({ items, tree }) => {
```

Comment:
Please consider a name that uses full words. I'm also not certain, based on this function name, what the function is intended to do. I don't think this function is getting used by anything yet (I think it was intended to be used `routes/items.js` to replace the repetitive code in the `/` and `/:item_id` routes.